command/server.go

```diff
@@ -1024,8 +1017,15 @@ CLUSTER_SYNTHESIS_COMPLETE:
 		c.UI.Warn("")
 		c.UI.Warn("You may need to set the following environment variable:")
 		c.UI.Warn("")
-		c.UI.Warn(fmt.Sprintf("    $ %s VAULT_ADDR=%s%s%s",
-			export, quote, "http://"+config.Listeners[0].Config["address"].(string), quote))
+
+		endpointURL := "http://"+config.Listeners[0].Config["address"].(string)
+		if runtime.GOOS == "windows" {
+			c.UI.Warn(fmt.Sprintf("    $env:VAULT_ADDR=\"%s\" (PowerShell)", endpointURL))
```

Comment:
In the past we've gotten complaints from people if they couldn't click to highlight the whole line and then copy and paste...can you move the PowerShell/cmd.exe bits to a line above instead of inline with VAULT_ADDR?

Reply:
Updated.